lib/handlers/notify.js

```diff
@@ -38,18 +38,32 @@ function getParentActivity (method, status) {
   return 'Update'
 }
 
+function filterMillseconds (isoDate) {
+  return `${isoDate.substring(0, 19)}${isoDate.substring(23)}`
+}
+
+function getDate (date) {
+  if (date) {
+    const eventDate = new Date(date)
+    if (!isNaN(eventDate.valueOf())) {
+      return filterMillseconds(eventDate.toISOString())
+    }
+  }
+  const now = new Date()
+  return filterMillseconds(now.toISOString())
+}
```

Comment:
Why filterMillseconds? Is it because that information is not actually available in value used for the Date / Last-Modified headers or it defaults to 000? (If elsewhere that information is available, it'd be preferable to not drop it.

Reply:
Because it will not just default to but always be .000. The reason being that the time the response that triggers the notification is sent is generated internally by Node and is exposed only via the Date header on the response. And sadly HTTP date header does not have millisecond resolution.